code/game/objects/items/stacks/medical_packs.dm

```diff
@@ -201,6 +203,7 @@
 	healverb = "salve"
 	heal_burn = 10
 	dynamic_icon_state = TRUE
+	parent_stack = TRUE
```

Comment:
no longer needed due to inheritance

Reply:
It should default to FALSE, as this is the item for the normal burn pack, its marked true to have its stacking handled separately then burn_pack/advanced.